src/main/java/com/cloudbees/jenkins/support/SupportLogFormatter.java

```diff
@@ -28,10 +28,12 @@
  * DO NOT INCLUDE ANY NON JDK CLASSES IN HERE.
  * IT CAN DEADLOCK REMOTING - SEE JENKINS-32622
  ***********************************************/
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
```

Comment:
@jtnord Is it ok to have a suppressFBWarning? I assume this should only impact compile time, but just want to check if this is ok.